core/src/components/item-sliding/test/async/item-sliding.e2e.ts

```diff
@@ -3,20 +3,37 @@ import { configs, test } from '@utils/test/playwright';
 
 configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
   test.describe(title('item-sliding: async'), () => {
-    test('should open even when item is added async', async ({ page }) => {
+    test.beforeEach(async ({ page }) => {
       await page.goto(`/src/components/item-sliding/test/async`, config);
+      const toggleButton = page.locator('#toggle-button');
 
-      const itemEl = page.locator('ion-item');
-      const itemSlidingEl = page.locator('ion-item-sliding');
+      await toggleButton.click();
+      await expect(toggleButton).toHaveClass(/hidden/); // class is added when everything is ready
+    });
 
-      // Wait for item to be added to DOM
-      await page.waitForSelector('ion-item');
+    test('should open even when item is added async', async ({ page }) => {
```

Comment:
Nit: Can we clarify the test titles? My understanding is the first one tests adding an `ion-item` async and the other one tests adding an `ion-item-options` async. However, the two titles are very similar so I was confused at first. Maybe using the full component name would help?